src/Optimization.jl

```diff
@@ -1,3 +1,29 @@
+struct MinimizationResult
+    converged :: Bool
+    iterations :: Int
+    data :: Optim.OptimizationResults
+end
+
+function Base.show(io::IO, ::MIME"text/plain", res::MinimizationResult)
+    (; converged, iterations, data) = res
+    Δx = number_to_simple_string(Optim.x_abschange(data); digits=2)
+    g_res = number_to_simple_string(Optim.g_residual(data); digits=2)
+    if converged
+        println(io, "Converged in $iterations iterations")
+    else
+        printstyled(io, "Non-converged", underline=true)
+        println(io, " after $iterations iterations: |Δx|=$Δx, |∂E/∂x|=$g_res")
```

Comment:
In the previous warning, we had also printed `|E|`, i.e. `Optim.f_abschange(data)`. That's the least useful of the three outputs, since `E` may be converged to tolerance $10^{-16}$ while `x` is only converged to tolerance $10^{-8}$. Either `|Δx|` or `|∂E/∂x|` is probably sufficient.